packages/address/src/address.ts

```diff
@@ -137,8 +136,9 @@ export default class Address extends AbstractAddress {
   }
 
   /**
-   * returns the address `checksum` as a string
-   *
+   * Returns the value of the `bech32Address` property
+   * @deprecated
+   * Type `Bech32Address` is now deprecated, as is this method. Use `B256` addresses instead. ([help](https://docs.fuel.network/docs/specs/abi/argument-encoding/#b256))
    * @returns The value of `bech32Address` property
    */
   override valueOf(): string {
```

Comment:
I don't think we will deprecate `valueOf`.

Reply:
Why would one use the `valueOf` method over the `toHexString` method for B256 addresses?

Reply:
Once we deprecate the `bech32Address` property won't `toHexString` also be redundant?

Reply:
@maschad I’m not sure why this was implemented initially, as we already had `toString` with the same functionality.

However, since this is no longer related to `bech32`, I believe it’s outside the scope of this PR and doesn’t need to be addressed here.